src/NmeaParser.Shared/Nmea/AltitudeUnit.cs

```diff
@@ -0,0 +1,40 @@
+﻿//
+// Copyright (c) 2016 Morten Nielsen
+//
+// Contributors:
+// Stephen Kennedy, Copyright (c) 2016 Gloucester Software Ltd.
```

Comment:
This gets unwieldy and unmaintainable. I'd rather have a consolidated Contributors.md file.